src/tools/merssh/command.cpp

```diff
@@ -276,18 +276,35 @@ void Command::maybeDoCMakePathMapping()
         }
 
         QString data = QString::fromUtf8(reader.data());
+
```

Comment:
I believe whitespace changes on unrelated lines in a same commit with functional changes are against Qt commit policy. Just saying, even though I'm not sure the Qt commit policy is really relevant here.

Reply:
Roughly speaking, I believe it makes crucial difference whether it is a change to existing non-blank lines or not. The reason for splitting changes into smaller portions is to allow determining the origin/purpose of code, and vertical space is rarely interesting in this respect. So I am OK with changes like this :)